### PR TITLE
add tidb_gogc_tuner_max_value and tidb_gogc_tuner_min_value

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/tidb/pull/47148

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -1840,6 +1840,26 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 默认值：`ON`
 - 该变量来用控制是否开启 GOGC Tuner。
 
+### `tidb_gogc_tuner_max_value` <span class="version-mark">从 v7.5.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline system-variables.md:1843]: To maintain consistency, please ensure all system variables are listed in alphabetical order. Specifically, position `tidb_gogc_tuner_max_value` and `tidb_gogc_tuner_min_value` ahead of `tidb_gogc_tuner_threshold`.

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -2551,6 +2551,26 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+### `tidb_gogc_tuner_min_value` <span class="version-mark">从 v7.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`100`
+- 范围：`[10, 2147483647]`
+- 该变量来用控制 GOGC Tuner 可调节 GOGC 的最小值。
```
Comment (Oreoxmt) [inline system-variables.md:2572]:
```suggestion
- 该变量用来控制 GOGC Tuner 可调节 GOGC 的最小值。
```

File: system-variables.md [hunk 3]
```diff
@@ -2551,6 +2551,26 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 
+### `tidb_gogc_tuner_max_value` <span class="version-mark">从 v7.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：整数型
+- 默认值：`500`
+- 范围：`[10, 2147483647]`
+- 该变量来用控制 GOGC Tuner 可调节 GOGC 的最大值。
```
Comment (Oreoxmt) [inline system-variables.md:2562]:
```suggestion
- 该变量用来控制 GOGC Tuner 可调节 GOGC 的最大值。
```

**Pull request merged**